test/optimizer/histogram_test.cpp

```diff
@@ -0,0 +1,24 @@
+#include "common/harness.h"
+
+#define private public
+
+#include "optimizer/stats/histogram.h"
+
+namespace peloton {
+namespace test {
+
+using namespace optimizer;
+
+class HistogramTests : public PelotonTest {};
+
+TEST_F(HistogramTests, SimpleHistogramTest) {
+  Histogram h{5};
+  for (int i = 1; i <= 10; i++) {
+    h.Update(i);
+  }
+  h.Print();
+  h.Uniform(4);
+}
```

Comment:
Instead of printing values in the test case, there should be EXPECT_EQ statements which check expected values. While the printing may be done initially while writing the test case, these should be later replaced with EXPECT statements.